ansible/vars/fedora/30.yml

```diff
@@ -6,4 +6,4 @@ repo_fedora_enabled: 1
 repo_updates_enabled: 1
 repo_updates_testing_enabled: 1
 repo_freeipa_copr_enabled: 1
-repo_pki_testing_enabled: 1
```

Comment:
Do not remove this, set it to zero instead, otherwise ansible complains:
```
TASK [builder/prepare : set default mock config] ***********************************************************************
fatal: [192.168.121.120]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'repo_pki_testing_enabled' is undefined"}
```
You also need to add `repo_pki_master_enabled: 0` to all other templates (except `rawhide`).